### ISSUE TITLE
Default link color failing WCAG2.0 AA contrast requirement on #f8f8f8 backgrounds

### ISSUE THREAD

**Issue by bradleywertman** [Esri/calcite-web#855]
After doing some initial accessibility testing on our sites using Calcite, we are noticing the default link color (#0079c1) does not have a high enough contrast ratio on elements with the background color of #f8f8f8. This looks to be mainly affecting the footers and the default panel color. 

[For WCAG2.0 AA](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html), the requirement is to have contrast ratio of 4.5:1. Currently the contrast ratio for these elements are 4.39:1. 

For the time being, to meet this WCAG requirement, we are thinking of overriding the default link color to #0074b8. Unless a new review of colors is coming soon ... RC9?

The last talk of color combinations and WCAG 2.0 looks to be from #755

**Comment by paulcpederson** [comment]
Yeah, a couple solutions come to mind:

1. Make the blue slightly darker
2. create a `link-dark-blue` modifier class which is meant specifically to sit on top of `$offwhite`

**Comment by paulcpederson** [comment]
Kind of related to #649 which was the same problem solved in a different way

**Comment by paulcpederson** [comment]
Next release will have a `link-dark-blue` class for a slightly darker link color (WCAG AA valid). Also adding a note about this in the documentation.
